connectors/byoc.py

```diff
@@ -758,6 +754,15 @@ async def sync(self, elastic_server, idling):
             await self._sync_done(job, {}, exception=e)
             raise
         finally:
+            if result is None:
+                result = {}
+            doc_updated = result.get("doc_updated", 0)
+            doc_created = result.get("doc_created", 0)
+            doc_deleted = result.get("doc_deleted", 0)
+            logger.info(
+                f"[{self.id}] Sync done: {doc_updated + doc_created} indexed, {doc_deleted} "
```

Comment:
```suggestion
                f"Sync done for connector with id [{self.id}]: {doc_updated + doc_created} indexed, {doc_deleted} "
```

Would suggest to explicitly state that the id belongs to a connector, otherwise this could cause confusion IMO.

Reply:
We have many tests relying on the format of this log, e.g:
https://github.com/elastic/connectors-python/blob/dbd82aa3b38351a870d32b07aceeac39090561eb/connectors/tests/test_sync.py#L386

Although I don't think this is the desired way of testing, I don't have plan to change it in this PR